pt/lessons/basics/date-time.md

```diff
@@ -88,20 +88,41 @@ iex> NaiveDateTime.add(~N[2018-10-01 00:00:14], 30)
 ## DateTime
 
 O segundo, como você pode ter adivinhado a partir do título dessa seção, é `DateTime`.
-Ele não tem as limitações vistas em `NaiveDateTime`: ele tem tanto o tempo e data, e suporta fuso horários.
-Mas note o comentário da documentação oficial sobre fuso horário:
+Não possui as limitações mencionadas no `NaiveDateTime`: possui data e hora e suporta fusos horários.
```

Comment:
```suggestion
Não possui as limitações vistas no `NaiveDateTime`: possui data e hora e suporta fusos horários.
```
I'm not sure on this one. But for me `vistas` seems to read a little bit better than `mencionadas`.

Reply:
@thiamsantos IMHO `vistas` or `mencionadas` it kind the same for me